src/cm.css

```
@@ -110,7 +117,12 @@ a:focus {
 .paragraph {
   color: var(--text-color);
   font-size: 14px;
-  margin: 0 0 18px 0;
+  margin: 0 0 1em 0;
+}
+
+/* Multiple paragraph interior styles */
```

```suggestion
/** Multiple paragraph interior styles */
```

parser only picks these up with double star